ghost/core/core/server/data/migrations/versions/5.55/2023-07-10-05-15-55-add-built-in-collections.js

```diff
@@ -0,0 +1,60 @@
+const logging = require('@tryghost/logging');
+const {default: ObjectID} = require('bson-objectid');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Creating built in collections');
+
+        const existingLatestCollection = await knex('collections')
+            .where({
+                slug: 'latest'
+            })
+            .first();
+
+        if (existingLatestCollection) {
+            logging.warn('Latest collection already exists, skipping');
+        } else {
+            await knex('collections').insert({
+                id: (new ObjectID()).toHexString(),
+                title: 'Latest',
+                slug: 'latest',
+                description: 'Collection with all posts',
+                type: 'automatic',
+                filter: '',
+                created_at: knex.raw('current_timestamp')
+            });
+        }
+
+        const existingFeaturedCollection = await knex('collections')
+            .where({
+                slug: 'featured'
+            })
+            .first();
+
+        if (existingFeaturedCollection) {
+            logging.warn('Featured collection already exists, skipping');
+        } else {
+            await knex('collections').insert({
+                id: (new ObjectID()).toHexString(),
+                title: 'Featured Posts',
```

Comment:
Could we standardise on not having `Posts` in the collection title? So `Latest` and `Featured` rather than `Latest` and `Featured Posts` so that when used in the UI it's consistent?